instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components'
+
+// Styled with Styled-Components
+const CommentDiv = styled.div`
+    text-align: left;
+    padding: 2px;
+    margin-left: 10px;
+`;
+
+const CommentUsername = styled.span`
+    font-weight: 500;
+    font-size: 16px;
+`;
+  
+const CommentText = styled.span`
+    font-weight: 300;
+    font-size: 18px;
+`;
+
+
+const Comment = props => {
+  return (
+    <CommentDiv>
+        <CommentUsername >{props.comment.username} </CommentUsername>
+      <CommentText >{props.comment.text}</CommentText>{' '}
+      
```

Comment:
Try to make functional components with the first letter as a lower case so const commentText but if its a class component use the first letter Uppercase it another great way to tell when you have a class or functional component